.github/workflows/autobuild.yml

```diff
@@ -62,15 +62,21 @@ jobs:
       publish_to_release:           ${{ steps.get-build-vars.outputs.PUBLISH_TO_RELEASE }}
       upload_url:                   ${{ steps.create-release.outputs.upload_url }}
       build_version:                ${{ steps.get-build-vars.outputs.BUILD_VERSION }}
+    env:
+      release_changelog_path:       ./.github_release_changelog.md
```

Comment:
Is this file in the .gitignore?

Reply:
Nice catch, thanks. The old name was and I had grepped for that, but missed hidden files. Updated .gitignore now.